src/blocks/shared/functions.scss

```
@@ -0,0 +1,12 @@
+@use "sass:math";
+@import "./variables";
+
+@function stripUnit($value) {
+
+	@return math.div($value, $value * 0 + 1);
+}
+
+@function to_rem($pxValue) {
+
+	@return math.div(stripUnit($pxValue), stripUnit($html-font-size)) + rem;
```

`stripUnit` not even needed since you are math.div ignores units

```suggestion
	@return math.div($pxValue, $html-font-size) + rem;
```

---

looking at the docs, i am probably wrong about this. but in testing it worked 🤷

---

`math.div` only ignores the unit if both numerator and denominator have same units (src: https://sass-lang.com/documentation/modules/math#div) and since this matches with our case so ignoring the `stripUnit` function.